plugin/cql/src/main/java/org/opencds/cqf/ruler/cql/CqlConfig.java

```diff
@@ -336,6 +337,11 @@ public ITermReadSvcR4 termReadSvcR4() {
 		return new PreExpandedTermReadSvcR4();
 	}
 
+	@Bean({ "artifactAssessment", "r4ArtifactAssessment", "r4ArtifactAssessment" })
+	@Conditional(OnR4Condition.class)
+	public ArtifactAssessment ArtifactAssessment() {
+		return new ArtifactAssessment();
+	}
```

Comment:
Not sure if this is required, intended to add ArtifactAssessment to the supported Resource types but wasn't able to make it work, currently ArtifactAssessments are created as Basic resources on `/Basic`